jbpm-wb-human-tasks/jbpm-wb-human-tasks-api/src/main/java/org/jbpm/workbench/ht/model/TaskDataSetConstants.java

```diff
@@ -43,6 +43,8 @@
     public static final String COLUMN_PROCESS_INSTANCE_CORRELATION_KEY = "correlationKey";
     public static final String COLUMN_PROCESS_INSTANCE_DESCRIPTION = "processInstanceDescription";
     public static final String COLUMN_EXCLUDED_OWNER = "entity_id";
+    public static final String COLUMN_SLA_DUE_DATE = "sla_due_date";
```

Comment:
Is it us, who define the column names? Please keep it consistent at least in the newly added columns. (Majority is camelCase so I'd stick to that) ...

Reply:
@jstastny-cz  this constants have to be consistent with the database column names to let the data set find the data at query. Then the answer is the value is defined by the database.